lib/tutorial.js

```diff
@@ -21,22 +21,22 @@ define([], function () {
 				<h3 class='popover-title tutorial-title'></h3>\
 				<div class='popover-content'></div>\
 				<div class='popover-navigation'>\
-					<div class='tutorial-prev-icon icon-button' data-role='prev'>\
+					<div align="left" class='tutorial-prev-icon icon-button' data-role='prev'>\
 						<div class='tutorial-prev-icon1 web-activity'>\
 							<div class='tutorial-prev-icon2 web-activity-icon'></div>\
 							<div class='tutorial-prev-icon3 web-activity-disable'></div>\
 						</div>\
 						<div class='icon-tutorial-text'>"+prevString+"</div>\
 					</div>\
 					<span data-role='separator'>|</span>\
-					<div class='tutorial-next-icon icon-button' data-role='next'>\
+					<div align="right" class='tutorial-next-icon icon-button' data-role='next'>\
```

Comment:
You can't add double-quote in a double-quoted string. It broke the syntax of the file.